packages/sideloader/build.sh

```diff
@@ -0,0 +1,43 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/Dadoum/Sideloader
+TERMUX_PKG_DESCRIPTION="Open-source cross-platform iOS app sideloader"
+TERMUX_PKG_LICENSE="GPL-3.0"
+_COMMIT=2e5c9670b592a2157c18b16244c3b82be838cd8a
+_COMMIT_DATE=20250223
+TERMUX_PKG_MAINTAINER="@xunmod"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_VERSION=1.0-pre4-p${_COMMIT_DATE}
+TERMUX_PKG_SRCURL=git+https://github.com/Dadoum/Sideloader
+TERMUX_PKG_SHA256=1e06c1c5d06cef55cb3a2c7731b12839b3d7cb368f2d059b8a11cb43707d5911
```

Comment:
This will break on the next commit.
`git+` source URLs do not generally include a SHA256 sum, they are instead pinned to the commit hash which acts as an alternate form of verification..